lib/tape/installer.rb

```diff
@@ -25,14 +25,28 @@ def install
         puts '🔎  Rails app detected'.pink
         copy_basic_examples
       end
-      copy_example 'templates/base/hosts.example', 'hosts'
-      mkdir 'dev_keys'
+      create_roles_dir
+      create_inventory_file
+      create_ssh_keys_dir
       print 'Are you going to use vagrant? (y/n): '
       if gets.chomp == 'y'
         copy_example 'Vagrantfile', 'Vagrantfile'
       end
     end
 
+    def create_roles_dir
+      mkdir "#{tapefiles_dir}/roles"
+      `touch #{tapefiles_dir}/roles/.keep`
+    end
+
+    def create_ssh_keys_dir
+      mkdir "#{tapefiles_dir}/dev_keys"
+    end
+
+    def create_inventory_file
+      copy_example 'templates/base/hosts.example', "#{tapefiles_dir}/hosts"
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.